### PR TITLE
Create OtherWaitResponse.go

### PR DESCRIPTION
**Pull request by Crenshaw1312**
Better for multiple questions, more user-friendly
-
it's not really a tutorial anymore, it's more of an actual command ig,so the name may need to change
Just read the notes at the top
I haven't found a good way to edit the ending message well

### CODE REVIEW

**Comment by Crenshaw1312** [comment]
Oh yea, this thing should be stable, but tbh I only had two stable runs before PRing it, So there may be like a couple bugs.

**Comment by Crenshaw1312** [comment]
I'm leaving the error thing as a define so ppl who know what they're doing can use errors easier when using this

File: Commands/ComplexCommands/TutorialWaitResponse/OtherWaitResponse.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+				{{$changeStage =1}}
+			{{ end }}
+		{{ else if and $num $databaseValue }}
+			{{ $responses = (cslice).AppendSlice (dbGet $.User.ID "responses").Value }}
+		{{ end }}
+
+{{/* Cancellation*/}}
+		{{ if and (eq (lower $.Message.Content) "cancel" "quit" "stop") $num }}
+			{{$embed.Set "title" "Cancelled"}}
+			{{$embed.Set "description" (print (or $.Member.Nick $.User.Username) "#" $.User.Discriminator " has decided to cancel the questionaire.")}}
```
Comment (DZ-TM) [inline Commands/ComplexCommands/TutorialWaitResponse/OtherWaitResponse.go]: erm... remove # discrim part... nickname with discrim is rather odd, altho (or .Member.Nick .User.String) is fine ig, altho I'd prefer to just keep username & nickname combo

Comment (Crenshaw1312) [reply Commands/ComplexCommands/TutorialWaitResponse/OtherWaitResponse.go]: Fixed

Comment (Crenshaw1312) [reply Commands/ComplexCommands/TutorialWaitResponse/OtherWaitResponse.go]: It's now just Username#Discrim

**Comment by Crenshaw1312** [comment]
*dust*

**Comment by DZ-TM** [comment]
_more dust_

**Comment by DZ-TM** [comment]
oof this is still open?

**Pull request closed without merging**